draft-packages/menu/KaizenDraft/Menu/Menu.tsx

```diff
@@ -5,9 +5,33 @@ import MenuDropdown from "./MenuDropdown"
 const styles = require("./styles.scss")
 
 export type GenericMenuProps = {
+  /**
+   * Whether the menu is to be used on the left or right
+   * side of the viewport. If left, the left of the dropdown
+   * is aligned to the left of the button (and vice versa)
+   * @default "left"
+   */
   align?: "left" | "right"
+
+  /**
+   * The width of the dropdown.
+   * "default": a fixed width of 248px
```

Comment:
Why 248px? Am I right in thinking that it was already doing this?

Reply:
Yeah, that was already there. Not sure why it was such as specific width, but I don't want to break anything existing.

Reply:
Yup, that makes sense.